app/scripts/views/settings/devices.js

```diff
@@ -75,7 +50,13 @@ define(function (require, exports, module) {
 
     _formatDevicesList: function (devices) {
       return _.map(devices, function (device) {
-        device.lastAccessTime = $.timeago(Number(device.lastAccessTime));
+        if (device.lastAccessTimeFormatted) {
+          device.lastAccessTime = Strings.interpolate(
+            t('Last active: %(translatedTimeAgo)s'), { translatedTimeAgo: device.lastAccessTimeFormatted });
```

Comment:
yeap. this does exactly that, except that all moment.js strings are already translated. and some of ours might not be. this might not be a huge problem